src/Services/ResponseTranslator.ts

```diff
@@ -46,10 +49,12 @@ export const defaultErrorResponseTranslator = (response: FetchResult<unknown> |
   const serviceErrorList: ServiceError[] = []
   if (isApolloErrorInternal(response)) {
     const { networkError, clientErrors, graphQLErrors, message } = response
-    if (networkError) {
+    if (networkError != null) {
       serviceErrorList.push({
         key: isServerParseError(networkError) ? ServiceErrorKey.CLIENT_ERROR : ServiceErrorKey.NETWORK_ERROR,
-        message: networkError.message || message,
+        message: (networkError.message.length > 0)
```

Comment:
are we sure that networkError.message != null?

Reply:
According to types `networkError` is `Error | ServerParseError | ServerError | null`.
Both `ServerParseError` and `ServerError` extend `Error` which has `message: string`, so it should be fine but I can add null check just in case.

Reply:
Yes let's add null check, we never now whot other error can be captured